src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -30,6 +30,16 @@ Optional<object> IExpression.ConstantValue
         SyntaxNode IOperation.Syntax => this.Syntax;
         
         protected virtual OperationKind ExpressionKind => OperationKind.None;
+
+        public virtual void Accept(IOperationVisitor visitor)
+        {
+            throw ExceptionUtilities.Unreachable;
+        }
+
+        public virtual TResult Accept<TResult>(IOperationVisitor<TResult> visitor)
+        {
+            throw ExceptionUtilities.Unreachable;
+        }
```

Comment:
Actually, maybe explicitness would improve maintainability in the long run. @JohnHamby Any thoughts?